### PR TITLE
Improved Docs

### PR DESCRIPTION
**Pull request by KaranGRohit**
Thanks for taking the time to contribute to Git!

Those seeking to contribute to the Git for Windows fork should see
http://gitforwindows.org/#contribute on how to contribute Windows specific
enhancements.

If your contribution is for the core Git functions and documentation
please be aware that the Git community does not use the github.com issues
or pull request mechanism for their contributions.

Instead, we use the Git mailing list (git@vger.kernel.org) for code and
documenatation submissions, code reviews, and bug reports. The
mailing list is plain text only (anything with HTML is sent directly
to the spam folder).

Nevertheless, you can use GitGitGadget (https://gitgitgadget.github.io/)
to conveniently send your Pull Requests commits to our mailing list.

Please read the "guidelines for contributing" linked above!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ fixes](https://github.com/git-for-windows/git/wiki/How-to-participate).
 Git - fast, scalable, distributed revision control system
 =========================================================
 
-Git is a fast, scalable, distributed revision control system with an
+Git is a fast, scalable, distributed revision/version control system with an
```
Comment (dscho) [inline README.md:21]: Not sure about that... in any case, this touches a part of the README that is not specific to Windows, and should therefore be sent to the Git mailing list (you can open a PR at https://github.com/gitgitgadget/git to do so).

File: README.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ CVS users may also want to read [Documentation/gitcvs-migration.txt][]
 (`man gitcvs-migration` or `git help cvs-migration` if git is
 installed).
 
-The user discussion and development of core Git take place on the Git
+The user discussion and development of core Git can take place on the Git
```
Comment (dscho) [inline README.md:47]: Or maybe replace "take" with "takes"?

File: README.md [hunk 3]
```diff
@@ -54,13 +54,13 @@ available at <https://lore.kernel.org/git/>,
 <http://marc.info/?l=git> and other archival sites.
 The core git mailing list is plain text (no HTML!).
 
-Issues which are security relevant should be disclosed privately to
+Issues which are security relevant and should be disclosed privately to
```
Comment (dscho) [inline README.md:57]: I find the original version is more understandable.

File: README.md [hunk 4]
```diff
@@ -54,13 +54,13 @@ available at <https://lore.kernel.org/git/>,
...
 the Git Security mailing list <git-security@googlegroups.com>.
 
 The maintainer frequently sends the "What's cooking" reports that
 list the current status of various development topics to the mailing
 list.  The discussion following them give a good reference for
-project status, development direction and remaining tasks.
+project status, development direction and the remaining tasks.
```
Comment (dscho) [inline README.md:63]: The original version sounds more open-ended, which reflects reality a bit better, I think.

**Review by dscho — COMMENTED** [review]
Hi! Thank you for working on the README.

I left a couple comments and suggestions, trying to help you improve the patch.

**Comment by keikoro** [comment]
@dscho This user is submitting nonsensical PRs to projects, likely in order to get a free Hacktoberfest t-shirt. Please consider labelling it as invalid and closing it.

**Pull request closed without merging**